README.md

```diff
@@ -33,3 +33,8 @@ This is a component that animates through an image sprite.
 - startFrame **{number}** - the frame to start animation
 - timeout **{number}** - the amount of millisecond between frame
 - stopLastFrame **{bool}** - stops animation from looping
+
+**Only required for two-dimensional sprites**
+
+- frameCount **{number}** - the total frame count of the sprite
```

Comment:
I like the implementation! What do you think about instead of `wrapAfter` we use something like `columnAmount` or `columns`?